app/views/events/show-partials/_intro.html.erb

```diff
@@ -1,10 +1,11 @@
+  
 <section id="intro" class="section-container">
   <div class="intro-container">  
     <div id="introCarousel" class="carousel slide carousel-fade" data-ride="carousel">
       <div class="carousel-inner" role="listbox">
         <div class="item active">
           <div class='carousel-background'>
-            <%= image_tag(serve_image_path(@event.logo)) %>
+            <%= image_tag(serve_image_path(@event_previews.first.image_link)) %>
```

Comment:
`serve_image_path` is used to serve images which are uploaded to the web directly, you can take a look at the routing in `config/routes.rb`. `@event_previews.first.image_link` is already in the form of link to image, so you can use `image_tag` directly. You can refer to the example in `app/views/gallery_photos/_gallery_photo.html.erb`.